shared-libs/cht-datasource/src/person.ts

```diff
@@ -26,6 +26,12 @@ export namespace v1 {
     readonly parent?: Place.v1.PlaceWithLineage | NormalizedParent,
   }
 
+  interface getPageParams {
```

Comment:
```suggestion
  interface GetPageParams {
```
make sure your interface name is capitalized